cypress/integration/countryClick_spec.js

```diff
@@ -0,0 +1,13 @@
+describe('Country Click test', () => {
+    it('successfully clicks & loads first, third and last country', () => {
```

Comment:
oh, maybe this test text was just copied :D test title is super important, i would say most important because this tells what you are testing inside